ci/.github/lint-filename.sh

```diff
@@ -9,7 +9,7 @@
 set -e
 
 SCRIPT_PATH=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
-GO_REGEX="^[a-zA-Z0-9_]+\.go$"
+GO_REGEX="^[a-zA-Z][a-zA-Z0-9_]?\.go$"
```

Comment:
`?` is equivalent to `{0,1}`.
```suggestion
GO_REGEX="^[a-zA-Z][a-zA-Z0-9_]*\.go$"
```

Reply:
Right you are, sorry 'bout that.